js/libs/keycloak-admin-client/src/utils/fetchWithError.ts

```diff
@@ -19,7 +21,8 @@ export async function fetchWithError(
 
   if (!response.ok) {
     const responseData = await parseResponse(response);
-    throw new NetworkError("Network response was not OK.", {
+    const description = getNetworkErrorMessage(responseData)
```

Comment:
Let's call this `message`, as that matches what the `Error` constructor expects.